src/changelog/sources/markdown/merger/merger.go

```diff
@@ -43,7 +43,7 @@ func New(ch *changelog.Changelog, newVersion *semver.Version) Merger {
 var (
 	unreleasedHeader = regexp.MustCompile(`(?i)^##\s*unreleased`)
 	heldHeader       = regexp.MustCompile(`(?i)^##\s*held`)
-	l2Header         = regexp.MustCompile(`^##\s*\w`)
+	l2Header         = regexp.MustCompile(`^##\s*[\w\[]`)
```

Comment:
I think essentially any character should work. In fact I think that was my original intention. What about
```suggestion
	l2Header         = regexp.MustCompile(`^##\s*\S`)
```